src/lib/services/classifiersService.ts

```diff
@@ -7,12 +8,32 @@ export interface ClassifiersData {
     brands: string[];
     colors: string[];
     priceIntervals: string[];
+    productVariants: ProductVariant[];
 }
```

Comment:
Perhaps this interface can be declared as a global one?

Reply:
I guess you right. Just to clarify - I only modified this file to display the data correctly, as it wasn't working as intended. I didn't actually create the file or the types (check the history).